config/routes.rb

```diff
@@ -454,13 +451,13 @@
     resources :work
 
     # legacy
-    get :chat, to: redirect('/chat/%{product_id}')
-    get :team, to: redirect(path: '%{product_id}/people')
-    get :welcome, to: redirect(path: '%{product_id}')
```

Comment:
Line is too long. [100/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.